MAINTAINERS.md

```diff
@@ -0,0 +1,79 @@
+  1. **features** All feature branches must be created under **feat/**,
+  2. **bug-fixes** All fixes must be created under **fix/**,
+  3. **chores** Ad-hoc tasks that are not features, minor housekeeping, maintenance tasks should be created under **chores/**
+  4. Avoid branches being grouped under your usernames
+  
+## II. Handling PRs
+
+- When creating a PR, you should:
+  1. Clearly describe the intent of the PR 
+  2. Describe the solution in detail with links to the original issue and any related issues that it might fix or close. 
+  3. GitHub Draft PRs are a great way to get CI or human feedback on work that isn't yet ready to merge. PRs can be created as drafts and converted to normal PRs once the CI passes. More information about GitHub Draft PRs: https://github.blog/2019-02-14-introducing-draft-pull-requests/
+  
+
+- PRs should be merged once they
+  1.  **pass the automated tests** (GitHub Actions, CLA signing, etc.),
+  2.  have the **review comments addressed**,
+  3.  get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
+  4.  have been open for at least **24 hours** unless the changes are trivial
+
+- To merge a pull request, it must have at least:
+
+  - one approval for simple documentation fixes
+  - two approvals for everything else
+
+- When merging a PR, you should:
+
+  1. Use the **merge strategy that produces a clean Git history**: ["Squash and merge"](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) commits and ensure the resulting commit message is:
+    - descriptive
+    - sentence case
+    - If instead the PR author took the time to craft individual, informative messages for each commit, then use the `Rebase and merge` method,to honor that work and preserve the history of the changes.
+    - For less clear-cut cases, a simple heuristic you can follow is that if there are more "dirty" commits than "clean" commits,then prefer squash, else do a rebase.
+
+  2. Update the [changelog](https://github.com/hirosystems/stacks-blockchain-api/blob/master/CHANGELOG.md) directly via the [GitHub website](https://github.com/stylelint/stylelint/edit/master/CHANGELOG.mdhttps://github.com/hirosystems/stacks-blockchain-api/edit/master/CHANGELOG.md)
+    1. Group the item under and in the following order: "Features", "Bug Fixes", "Reverts", "Chores", "Documentation", 
+    2. All Breaking Changes **must** be listed at the top of the changelog under "⚠ BREAKING CHANGES", followed by any "Compatibility Issues" or "Known Limitations". This section serves as a notification that this isn't a straightforward upgrade.
+    3. Order the item within the group by the most-significant first to the smallest, and then alphabetically by item name.
+    4. Suffix the item with the relevant issue number and pull request number, 
+  3. Post this update as a comment to the pull request.
```

Comment:
The changelog file is [updated automatically](https://github.com/hirosystems/stacks-blockchain-api/commits/master/CHANGELOG.md) by the semantic-release bot. It pulls the data from commit messages using conventional commits.

The only bit of manual work related to this is: we sometimes clean up the automated release messages (i.e. these https://github.com/hirosystems/stacks-blockchain-api/releases) for clarity and any important information related to upgrading (if needed). We probably don't need to mention that here, because it's on a case-by-case basis and often isn't required.